src/app/firebase_app.ts

```diff
@@ -288,32 +285,42 @@ class FirebaseAppImpl implements FirebaseApp {
   /**
    * Return the service instance associated with this app (creating it
    * on demand).
+   * @internal
    */
-  private getService(name: string, instanceString?: string): FirebaseService
```

Comment:
This function wasn't removed, but renamed. With the new registration methodology, we need to call this function outside of the class which is prevented by the `private` keyword.

It was replaced with the internal `_getService` function that we call both inside of the class and outside.